cumulusci/core/config/project_config.py

```diff
@@ -42,7 +43,23 @@
 )
 
 
-class BaseProjectConfig(BaseTaskFlowConfig):
+class ProjectConfigPropertiesMixin(BaseConfig):
+    cli: dict
+    services: dict
+    project: dict
+    cumulusci: dict
+    orgs: dict
+    minimum_cumulusci_version: str
+    sources: dict
+    flows: dict
+    plans: dict
+    tasks: dict
+    repo_owner: str
+    repo_name: str
+    dev_config: dict  # this is not documented and should be deprecated
```

Comment:
Where is it used?

Reply:
Can that use .lookup() instead?

Reply:
It can, but why treat it differently from everything else? It's a valid property which should either be deprecated or documented and hiding it in the code will delay rather than accelerate that process.